CMakeLists.txt

```diff
@@ -35,6 +35,9 @@ project(Csound)
 
 enable_testing()
 
+set(libcsound_private_include_dirs "")
+set(libcsound_public_include_dirs "")
```

Comment:
Should these be upper case just to be consistent? Could probably just be `LIBCSOUND_PRIVATE_INCLUDES` etc.